packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_login/ui_components/login_form.dart

```diff
@@ -73,6 +77,15 @@ class _LoginFormState extends State<LoginForm> {
                   ),
                   child: _buildLogInButton(),
                 ),
+                {{#enable_feature_google_login}}
+                const Divider(indent: 5, endIndent: 5),
+              Padding(
+                  padding: EdgeInsets.symmetric(
+                    horizontal: context.designSystem.spacing.l,
+                  ),
+                  child: const LibAuthGooglePageWithDependencies(),
+                ),
+                {{/enable_feature_google_login}}
```

Comment:
Having the button inside the login form and making it looks similar to the main LogIn buttom may confuse users. Please, move the button outside the form.